node-red-container/Dockerfile-4.0

```
@@ -0,0 +1,35 @@
+FROM nodered/node-red-dev:4.0.0-beta.1-18
```

```suggestion
FROM nodered/node-red-dev:4.0.0-beta.1
```

We should be using the NodeJS v20 build not the NodeJS v18